kotest-assertions/kotest-assertions-core/src/jvmTest/kotlin/com/sksamuel/kotest/matchers/collections/SequenceMatchersTest.kt

```diff
@@ -431,12 +433,16 @@ class SequenceMatchersTest : WordSpec() {
       }
 
       "have element at" should {
-         abort<IndexOutOfBoundsException>("for empty") {
-            sampleData.empty.shouldHaveElementAt(sampleData.empty.count(), 0)
+         "handle empty sequence" {
+            shouldThrow<AssertionError> {
+               sampleData.empty.shouldHaveElementAt(sampleData.empty.count(), 0)
+            }.message shouldContain "but the sequence was empty"
```

Comment:
🔵 I see this more as a bugfix than a breaking change. I think we are fine on that front